### PR TITLE
Illustrated stories  -Adding Bibi sara story and other stories

### PR DESCRIPTION
**Pull request by maxwellfundi**
@michaelclapham  i went through the process of adding the illustrated stories and i think i got it all right now. 
Please have a look and let me know what you think. One question is how to test an individual component of the app, because the  button on activities on stories doesn't work really.

### CODE REVIEW

**Comment by maxwellfundi** [comment]
@michaelclapham  another concern i have about these illustrated stories is there is such a huge spce left at the bottom of the screen on the app. I wonder what we can do about it, probably add something about emotions?

**Comment by maxwellfundi** [comment]
@michaelclapham  i have now added the images in the jpg format. This can be merged whilst i continue to add the others.

**Comment by michaelclapham** [comment]
> @michaelclapham i have now added the images in the jpg format. This can be merged whilst i continue to add the others.

Do you want to keep adding them on this branch?

**Comment by maxwellfundi** [comment]
I would like this to be merged. I will add more on a different branch.

File: src/app/stories/bibi-sara-story/bibi-sara-story.component.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<p>Bibi-sara-story works!</p>
+
+<ion-content>
```
Comment (shiundumichael) [inline src/app/stories/bibi-sara-story/bibi-sara-story.component.html:3]: @michaelclapham @maxwellfundi  I dont understand if this is an html tag for i have never seen it before

**Pull request merged**